modules/sb-about-template-service-broker.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-template-service-broker.adoc
+
+[id='sb-about-template-service-broker-{context}']
+= About the template service broker
```

Comment:
@shawn-hurley Can you take a look at this summary of the template service broker (brought forward from 3.x) that I added in before getting in to installing it and see if it's good enough for 4.0 or if anything should be changed?

http://file.rdu.redhat.com/~ahoffer/2019/OSDOCS-176-update-installing-sb/applications/service_brokers/installing-template-service-broker.html